packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -38,6 +38,7 @@
 
 	--border-color-error: #{kv-color('error')};
 	--right-slot-width: 36px;
+	--left-slot-width: #{$spacing-6x};
```

Comment:
Should this be padding? 
If so, please change the name to `--left-slot-padding`.
Otherwise, it doesn't make much sense to use our spacing system for the width property, so simply change the value 24px

Reply:
Sure! makes sense! will change